portfolio/src/main/webapp/script.js

```diff
@@ -12,12 +12,17 @@
 // limitations under the License.
 
 
-fetch('/data').then(response => response.json()).then((comment) => {
-	let doc = document.getElementById('comment-container');
-	for(x in comment){
-		doc.innerHTML += "<li>";
-		doc.innerHTML += "<h3>" + comment[x].name + "</h3>";
-		doc.innerHTML += "<p>" + comment[x].content + "</p>";
-		doc.innerHTML += "</li>";
+fetch('/data').then(response => response.json()).then((comments) => {
+	let commentSection = document.getElementById('comments-container');
+    
+	for(x in comments){
```

Comment:
If `comments` is an Array, we should use [`for...of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) instead of [`for...in`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in). For one thing, `for...in` doesn't guarantee the order of items returned, so it may undo any ordering of comments you've prepared on the backend.

It will give you each element of the array as your loop variable (`x` currently, but we probably want a more descriptive name) and you can hand that off to `addComment`.